frontend/src/components/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import {
+    BrowserRouter as Router,
+    Route,
+    Link
+  } from 'react-router-dom'
+
+  import MainPageContent from './MainPageContent';
+  import HelloWorldPage from './HelloWorldPage';
+  import LoginPageContent from './LoginPageContent';
+
+const App = () => (
+        <div>
```

Comment:
When Reacts requires you to have a single root element, don't use meaningless divs that pollute your markup. Use react fragments feature.